src/pharmpy/modeling/eta_additions.py

```diff
@@ -133,6 +138,11 @@ def add_iov(model, occ, list_of_parameters=None, eta_names=None):
     eta_names : str, list
         Custom names of new etas. Must be equal to the number of input etas times the number of
         categories for occasion.
+    distribution : str
+        The distribution that should be used for the new etas. Options are
+        'disjoint' for disjoint normal distributions, 'joint' for joint normal
+        distribution, 'mixed' for an explicit mix of joint and disjoint
```

Comment:
Is `'mixed'` really a good name? I think I was influenced by some unrelated use of `'mixed'` in existing code.